core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -67,24 +78,35 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
     producer.close(60, TimeUnit.SECONDS)
   }
 
+  def produceMessages(topic: String, range: Range) = {
+    Source(range)
```

Comment:
If you plan to use it in synchronous manner every time then lets put `Await` inside this function